ci-macos.yml

```diff
@@ -25,8 +25,9 @@ builders:
   folder: "In Progress Base VMs"
   template: "travis-ci-macos10.13-xcode9.4-vanilla"
   vm_name: "{{ user `image_name` }}"
-  CPUs: 2
+  CPUs: 4
```

Comment:
Are we beefing it up for the duration of the build, and then intending on creating a 2-CPU VM afterwards? Just want to make sure that that's what's up, because it would be a pain for the step to move it back down to 2 CPUs to be missed, and then severely overcommitting our compute capacity.

Reply:
Yes, that's the intention here! It should only be 4 CPUs during the build. I can update the builders manual to include that step, and I hope we can eventually automate it somehow.

Reply:
Hi!

I think this scenario is a legit concern and I'd like to ask for this to either be entirely automated (i.e.both  increasing and decreasing CPUs) or not made part of automation right now. I have questions about whether a mostly-automated process is going to get a careful reading of the docs every time over years, especially given that the docs can move in and out of close accuracy depending on team workload, which corresponds to their having greater and less credibility with the reader.

Let's make figuring out how to automate this entirely a prerequisite for adding it permanently – fine to hold off on this until Monday to discuss more with me 1:1 if that has complexities. Thank you!

Reply:
^^ I planned to do some vsphere-related work today to be able to make a 12 core 64GB RAM macpro available exclusively for packer image development (via cluster paths) so we can get rid of some of the pain of long feedback loops. 

The issue: the packer build image process takes forever; feedback loops are hours-long wait, network failures mean we need to start all over again.  
One solution:  give the packer build VM more cores/RAM to do the work faster, so we get a quicker feedback loop (instead of waiting hours / a whole day). 
How to implement: take 1 macpro from the `Main_MacPro_pod_1` cluster, add it to a new `Packer-Image-Dev` (or some name) cluster, point the `ci-macos.yml` to that path/cluster, we can now use up to 12 core / 64 GB RAM to see if this makes a difference in the feedback loop time.

I do however, want to keep in mind Halle’s comment here about having to change hardware resources and its consequences. 

I'll hold off on implementing this, and we can factor in a dedicated macpro as part of the automation solution.

Reply:
Thank you for your care on that question! I don’t see investigation of getting past this automation pain as time ill-spent (I think the goal in the case of our talk earlier this week was to make it painless to switch a single Mac Pro from packer work specs to build job specs and vice versa), as long as it is timeboxed versus other important things, and nothing is postponing fixing the 9.4.1 image release issues and completing the entire testbed for testing the packer script images so we can see if there are other issues. If specs need to be turned up just for that work this week, it's fine, with the proviso that that shouldn't end up in trunk until we have automation for the full lifecycle of spec changes, and responsibility remains with the spec-turner-upper to make sure it isn't permanent. Thanks!

Reply:
That makes sense, @Halle. I'll remove the 4 CPU change from this PR then, and we can add it back when we have measures to ensure it doesn't stay that way.

Reply:
Awesome, thanks all.